src/scenes/home/team/team.js

```diff
@@ -31,6 +31,7 @@ class Team extends Component {
 
     const boardMembers = BoardMembers.map(boardMember => (
       <BoardCard
+        key={`${Math.random()} + ${boardMember.name}`}
```

Comment:
It's a huge nitpick, but we can simply use ${boardMember.name} + ${boardMember.role} inside key to avoid a function call.